src/main/java/com/radixdlt/DefaultSerialization.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
+ *  *
+ *  * Radix DLT Ltd licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except in
+ *  * compliance with the License.  You may obtain a copy of the
+ *  * License at
+ *  *
+ *  *  http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  * either express or implied.  See the License for the specific
+ *  * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt;
+
+import com.radixdlt.serialization.Serialization;
+import com.radixdlt.serialization.core.ClasspathScanningSerializationPolicy;
+import com.radixdlt.serialization.core.ClasspathScanningSerializerIds;
+
+public final class DefaultSerialization {
+
+	private DefaultSerialization() {
+		//not called
+	}
+
+	private static final Serialization defaultInstance = Serialization.create(ClasspathScanningSerializerIds.create(),
+			ClasspathScanningSerializationPolicy.create());
+
+	public static Serialization getDefault() {
+		return defaultInstance;
```

Comment:
Would be better implemented using Holder initialisation pattern.

https://en.wikipedia.org/wiki/Initialization-on-demand_holder_idiom

Reply:
fixed